### ISSUE TITLE
Web Resource Deployment - Profiles

### ISSUE THREAD

**Issue by aelsmore** [scottdurow/SparkleXrm#224]
Hi Scott

As the only way of deploying individual webresources is to create a separate profile containing them... what do you think about adding a "addtodefaultprofile" flag to the webresources section, that adds the webresource json to the detault profile section (if not already exists) after deployment?

Cheers
Andy

**Comment by scottdurow** [comment]
Interesting idea - so the specific challenge is that when you create a new web resource in your project, you don't want to run the deployment for all - just the new one?

Other options might be having a flag on the deploy task that includes a set of glob paths to deploy.
Or perhaps some kind of checksum to see which have changed or are new and only deploy those.

**Comment by aelsmore** [comment]
Yes. So the use case would be, working in a patch solution as part of a sprint. You wouldn't want to deploy everything to your patch, but you would want the webresource to eventually end up in your main solution once it is upgraded.

![image](https://user-images.githubusercontent.com/6952441/38977012-ef2a13dc-43aa-11e8-940f-b9e801495965.png)

**Comment by aelsmore** [comment]
I copy and pasted the main solution components for effect... I wouldn't have any duplicates in reality!
